permissions/plugin-h2-api.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "h2-api"
+github: "jenkinsci/h2-api-plugin"
+paths:
+- "org/jenkins-ci/plugins/h2-api"
```

Comment:
@cyrille-leclerc note that most modern plugins now use `io.jenkins.plugins` instead of the old groupId. 

We try to remind this to new requests generally, even if some go through because we miss them.

Reply:
@slide about that, is this something that would be possible to add in your new plugin checker? thanks!

Reply:
Yes, I can add a check for that.

Reply:
@batmat, are you wanting to hold off on this for the groupId change? I don't think it's a blocker myself.

Reply:
Nope not a blocker, I still approved it :).

Reply:
change made: https://github.com/jenkinsci/h2-api-plugin/commit/43f45570ad1059f1fa74ce6a0d6cbf7814f44b06